### PR TITLE
[DE] Add `Beatmap/Genre and language`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
The two genres `hip hop` and `rock` link Wikipedia articles in their description. I had a quarrel on how to proceed with those while translating. Originally, I saw these options:

1. translate literally and link to english article
2. translate literally and link to german article
3. translate in free form

The third option is very likely not wanted and should only be used as a backup.

While both Wikipedia articles[^wiki] essentially say the same, they do it with very different words (see the third option) and I'm not sure if the second option makes sense for something which is meant as a citation. Despite this, I still decided to go for this option, because it is probably the most user friendly one.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

[^wiki]: I'm refering to the German and English articles for each genre.

### CODE REVIEW

File: wiki/Beatmap/Genre_and_language/de.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+| Anime | Produziert für oder bekannt geworden durch einen Anime oder ein vergleichbares Medium. |
+| Rock | Mit einem Schwerpunkt auf Gitarren, Schlagzeug und Bass, typischerweise auf der Grundlage von einfachen synkopischen Rhythmen basierend und von Live-Auftritten und einem Fokus auf ernste sowie progressive Motive charakterisiert ([*Rockmusik* auf Wikipedia](https://de.wikipedia.org/wiki/Rockmusik)). |
+| Pop | In der Popkultur weit verbreitet und durch die Verwendung einer einfachen Struktur und wiederholter Refrains oder Melodien eingängig komponiert. |
+| Andere | Keinem der aufgeführten Genres zuzuordnen. |
+| Novelty | Lustig oder ungewöhnlich. In osu! beinhaltet diese Kategorie hauptsächlich Songs wie "YouTube Poops", "Niconico MADs" und Meme-Musik. |
+| Hip-Hop | Stilisierte rhythmische Musik, die häufig das Rappen begleitet und manchmal Samples oder Bassaufnahmen verwendet ([*Hip-Hop* auf Wikipedia](https://de.wikipedia.org/wiki/Hip-Hop)). |
+| Electronic | Digital oder mit elektronischen Instrumenten komponiert. |
+| Metal | Ausdrucksstarker Sound mit lauten, verzerrten E-Gitarren und Bässen sowie schnellen oder rasanten Schlagzeugrhythmen. |
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Genre_and_language/de.md:47]: I'm not sure if `dichter Rhythmus` (dense rhythm) is something a native would say.

File: wiki/Beatmap/Genre_and_language/de.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+| Klassik | Formal, auf der Grundlage fortgeschrittener Musiktheorie strukturiert und in der alten westlichen Kultur verwurzelt oder von ihr inspiriert. |
+| Folk | Traditionell oder persönlich, leger und die Identität einer Kultur oder einer Gruppe von Menschen widerspiegelnd. |
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Genre_and_language/de.md:49]: If `leger` is something you find too uncommon, `locker` might be an alternative.

File: wiki/Beatmap/Genre_and_language/de.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Eine Beatmap kann nicht [nominiert](/wiki/Beatmap_ranking_procedure#nominations) werden, wenn die Sprache oder das Genre auf `Nicht angegeben` eingestellt ist.
+
+## Liste der Genres
+
+| Genre | Merkmale entsprechender Songs |
+| :-- | :-- |
+| Nicht angegeben | Keine Songs treffen zu. Dieses Genre ist ein Platzhalter, bis ein geeignetes Genre festgelegt ist. |
+| Videospiel | Gemacht für oder populär geworden durch ein Videospiel. Dies wird auch für Arrangements und Remixes dieser Songs verwendet. |
+| Anime | Produziert für oder bekannt geworden durch einen Anime oder ein vergleichbares Medium. |
+| Rock | Mit einem Schwerpunkt auf Gitarren, Schlagzeug und Bass, typischerweise auf der Grundlage von einfachen synkopischen Rhythmen basierend und von Live-Auftritten und einem Fokus auf ernste sowie progressive Motive charakterisiert ([*Rockmusik* auf Wikipedia](https://de.wikipedia.org/wiki/Rockmusik)). |
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Genre_and_language/de.md:41]: Keep in mind that the citation used here is incorrect. It says "unsyncopated", but in the Wikipedia article, it says "syncopated".

File: wiki/Beatmap/Genre_and_language/de.md (hunk 1 above)
Comment (respektive) [reply wiki/Beatmap/Genre_and_language/de.md:47]: Yeah I don't know either if `dichter Rhythmus` is something people say, but I think your translation is fine for this instead.

File: wiki/Beatmap/Genre_and_language/de.md (hunk 2 above)
Comment (respektive) [reply wiki/Beatmap/Genre_and_language/de.md:49]: I think `locker` works probably better for this. I don't think I ever heard `leger` in this context.

**Review by respektive — CHANGES_REQUESTED** [review]
About the Wikipedia articles, I think just linking to the German articles is the best option yeah.

File: wiki/Beatmap/Genre_and_language/de.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+tags:
+  - genres
+  - languages
+  - metadata
+  - Sprachen
+---
+
+# Genre und Sprache
+
+![Ein Screenshot des Genres, der Sprache und die Tags auf einer Beatmap-Seite](img/beatmap-info-DE.png "Genre und Sprache werden auf Beatmap-Seiten angezeigt.")
```
Comment (FreezyLemon) [inline wiki/Beatmap/Genre_and_language/de.md:11]: Either `... und der Tags ...` or `Ein Screenshot von Genre, Sprache und Tags ...`

File: wiki/Beatmap/Genre_and_language/de.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+## Beatmap-Auflistung
+
+![Die verfügbaren Filter in der Beatmap-Auflistung](img/beatmap-listing-DE.png)
+
+Die [Beatmap-Auflistung](https://osu.ppy.sh/beatmapsets) hat Filter für das Genre und die Sprache im Dropdown-Menü `Mehr Suchoptionen`.
+
+## Das Genre und die Sprache einer Beatmap ändern
+
+Das Genre und die Sprache kann auf der Webseite geändert werden, indem man mit dem Mauszeiger über die Details einer Beatmap fährt und dabei auf das Bleistiftsymbol klickt. Dabei bestimmen die [Benutzergruppe](/wiki/People/User_group) des Nutzers und die [Kategorie](/wiki/Beatmap/Category#aktuelle-kategorien) der Beatmap die jeweiligen Rechte:
```
Comment (FreezyLemon) [inline wiki/Beatmap/Genre_and_language/de.md:25]: `Das Genre und die Sprache können`

Comment (FreezyLemon) [reply wiki/Beatmap/Genre_and_language/de.md:25]: Maybe `... fährt und auf ...` (remove the `dabei`)

File: wiki/Beatmap/Genre_and_language/de.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+- Beatmap-Eigentümer dürfen das, wenn sich die Beatmap in der Kategorie [WIP](/wiki/Beatmap/Category#work-in-progress-und-ausstehend), [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) oder [Friedhof](/wiki/Beatmap/Category#friedhof) befindet und keine [Nominierungen](/wiki/Beatmap_ranking_procedure#nominations) hat.
```
Comment (FreezyLemon) [inline wiki/Beatmap/Genre_and_language/de.md:27]: `dürfen das, wenn...` in this section doesn't read great. I see that the structure of the english article makes it a bit difficult to translate...

Maybe `können Änderungen vornehmen, wenn...`? It's more verbose though.

File: wiki/Beatmap/Genre_and_language/de.md (hunk 1 above)
Comment (FreezyLemon) [reply wiki/Beatmap/Genre_and_language/de.md:47]: "rasant" seems redundant here. I'd just leave it at "schnellen Schlagzeugrhythmen".

File: wiki/Beatmap/Genre_and_language/de.md (hunk 2 above)
Comment (FreezyLemon) [reply wiki/Beatmap/Genre_and_language/de.md:49]: I prefer `locker` too. Maybe add a comma after `leger`/`locker` (whichever you'll use).

File: wiki/Beatmap/Genre_and_language/de.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+| Andere | Keinem der aufgeführten Genres zuzuordnen. |
+| Novelty | Lustig oder ungewöhnlich. In osu! beinhaltet diese Kategorie hauptsächlich Songs wie "YouTube Poops", "Niconico MADs" und Meme-Musik. |
+| Hip-Hop | Stilisierte rhythmische Musik, die häufig das Rappen begleitet und manchmal Samples oder Bassaufnahmen verwendet ([*Hip-Hop* auf Wikipedia](https://de.wikipedia.org/wiki/Hip-Hop)). |
+| Electronic | Digital oder mit elektronischen Instrumenten komponiert. |
+| Metal | Ausdrucksstarker Sound mit lauten, verzerrten E-Gitarren und Bässen sowie schnellen oder rasanten Schlagzeugrhythmen. |
+| Klassik | Formal, auf der Grundlage fortgeschrittener Musiktheorie strukturiert und in der alten westlichen Kultur verwurzelt oder von ihr inspiriert. |
+| Folk | Traditionell oder persönlich, leger und die Identität einer Kultur oder einer Gruppe von Menschen widerspiegelnd. |
+| Jazz | Der Schwerpunkt liegt auf Improvisation und das Entlehnen afrikanischer und kubanischer Rhythmen. Enthält typischerweise Pianos und Instrumente aus Marschkapellen. |
```
Comment (FreezyLemon) [inline wiki/Beatmap/Genre_and_language/de.md:50]: `und dem Entlehnen`

**Review by FreezyLemon — CHANGES_REQUESTED** [review]
Nice work. I left some suggestions and nitpicks.

Regarding the Wikipedia articles, I think you can either do option 2 or try and find some descriptions of the music genre in the german wikipedia articles and quote those instead (maybe that's what you meant by option 3). I don't think deviating from the english version is a problem in this instance.

File: wiki/Beatmap/Genre_and_language/de.md (hunk 4 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md:11]:
```suggestion
![Ein Screenshot des Genres, der Sprache und der Tags auf einer Beatmap-Seite](img/beatmap-info-DE.png "Genre und Sprache werden auf Beatmap-Seiten angezeigt.")
```

File: wiki/Beatmap/Genre_and_language/de.md (hunk 5 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md:25]:
```suggestion
Das Genre und die Sprache können auf der Webseite geändert werden, indem man mit dem Mauszeiger über die Details einer Beatmap fährt und auf das Bleistiftsymbol klickt. Dabei bestimmen die [Benutzergruppe](/wiki/People/User_group) des Nutzers und die [Kategorie](/wiki/Beatmap/Category#aktuelle-kategorien) der Beatmap die jeweiligen Rechte:
```

File: wiki/Beatmap/Genre_and_language/de.md (hunk 6 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md:27]: Yeah, good point. I second guessed this part too. I will take another look at it.

File: wiki/Beatmap/Genre_and_language/de.md (hunk 3 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md:41]: Closing this note due to #10117 already being merged.

File: wiki/Beatmap/Genre_and_language/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md:47]:
```suggestion
| Metal | Ausdrucksstarker Sound mit lauten, verzerrten E-Gitarren und Bässen sowie schnellen Schlagzeugrhythmen. |
```

File: wiki/Beatmap/Genre_and_language/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md:49]:
```suggestion
| Folk | Traditionell oder persönlich, locker und die Identität einer Kultur oder einer Gruppe von Menschen widerspiegelnd. |
```

I don't think a comma should be put there.

File: wiki/Beatmap/Genre_and_language/de.md (hunk 7 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md:50]:
```suggestion
| Jazz | Der Schwerpunkt liegt auf Improvisation und dem Entlehnen afrikanischer und kubanischer Rhythmen. Enthält typischerweise Pianos und Instrumente aus Marschkapellen. |
```

File: wiki/Beatmap/Genre_and_language/de.md (hunk 6 above)
Comment (The-Last-Cookie) [reply wiki/Beatmap/Genre_and_language/de.md]: 1f5018b9e776a78fd7ab7078cd78b992615b46f4

**Review by FreezyLemon — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**